Open Judge System/OJS.Common/ObjectFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OJS.Common
+{
+    using System;
+
+    using Extensions;
+
+    public static class ObjectFactory
```

Comment:
In what kind of cases do we **need** to use an `ObjectFactory` ? Can't we make use of `Ninject`'s DI?

Reply:
I need an instance of the `IContestsDataService` in one method of the `BaseController`, where it is not a good idea to inject it from the constructor.